definitions/npm/@zeit/title_v3.4.x/flow_v0.30.x-/title_v3.4.x.js

```diff
@@ -0,0 +1,7 @@
+declare module '@zeit/title' {
+  declare type Options = $Exact<{
+    specials: Array<string>,
```

Comment:
Why $Exact and not `{||}`? Specials could perhaps be immutable `+specials`, as well as the type could be `$ReadOnlyArray`?

Reply:
> Why $Exact and not `{||}`
Not real reason, but I was thinking, isn't it a bit more "future proof" when they will remove the `{||}` notation?

Updating the rest, nice suggestions!

Reply:
There will be codemod to get rid of it. It needs to be run against the whole repo neverthless. IT at least removes the pipes, not sure about $Exact, see https://github.com/flow-typed/flow-typed/issues/2876